### PR TITLE
feat(bpmn): add zoom methods and simulator

### PR DESCRIPTION
**Pull request by jessevanmuijden**
https://github.com/biz-dev-ops/web-components/issues/93

### CODE REVIEW

**Comment by arjangeertsema** [comment]
@jessevanmuijden can you merge main into this branch? I have tried, but there are a lot of conflicts and it did not work. Then I will do a final check before merging this pull request.

File: src/bpmn-viewer/test-data/test_subprocesses.bpmn.ts
Comment (arjangeertsema) [inline src/bpmn-viewer/test-data/test_subprocesses.bpmn.ts:1]: @jessevanmuijden can you link the data-xml attribute value to this test data in the bpmn viewer index.html file?

Comment (arjangeertsema) [reply src/bpmn-viewer/test-data/test_subprocesses.bpmn.ts:1]: @jessevanmuijden can you also move the css from the bpmn viewer index.html file to a separate css file and use the font face from the bpmn library?

Comment (arjangeertsema) [reply src/bpmn-viewer/test-data/test_subprocesses.bpmn.ts:1]: @jessevanmuijden is it possible to exclude the test files and html files from the actual deployment to npm to safe space?

Comment (arjangeertsema) [reply src/bpmn-viewer/test-data/test_subprocesses.bpmn.ts:1]: @jessevanmuijden can you add simpel zoom buttons to bpmn viewer index.html which link to the api of the component? (the html files should explain how te component should be used = documentation)

**Review by arjangeertsema — CHANGES_REQUESTED** [review]

File: src/bpmn-viewer/test-data/test_subprocesses.bpmn.ts
Comment (jessevanmuijden) [reply src/bpmn-viewer/test-data/test_subprocesses.bpmn.ts:1]: i addressed all issue. I would like to move the npm publication optimization to a separate issue/PR.

**Pull request merged**